rdflib/graph.py

```diff
@@ -1609,10 +1609,9 @@ def update(
 
         return processor.update(update_object, initBindings, initNs, **kwargs)
 
-    def n3(self) -> str:
+    def n3(self, namespace_manager: Optional["NamespaceManager"] = None) -> str:
```

Comment:
I'm a little confused.  The body of this method is updated to use `namespace_manager` as a keyword argument on an `n3()` method; but, this syntax looks like it's specifying `namespace_manager` as a positional argument.

Could the signature become instead:

```python
def n3(
    self,
    *args: Any,
    namespace_manager: Optional["NamespaceManager"]=None,
    **kwargs: Any,
) -> str: ...
```